server.js

```diff
@@ -1,3 +1,14 @@
 import app from './app.js';
+import { connectToDB } from './connections.js';
 
-app.listen({ port: process.env.PORT || 3000 });
+function loaders() {
+  try {
+    connectToDB();
+    app.listen({ port: process.env.PORT || 3000 });
+  } catch (error) {
+    console.log(error);
+    throw error;
```

Comment:
now it the time to handle the error, because there is no wrapper:)

Reply:
done

Reply:
so don't throw it because no body listen :)